src/pages/settings/Profile/ProfilePage.js

```diff
@@ -290,8 +290,8 @@ class ProfilePage extends Component {
                             label={this.props.translate('profilePage.timezone')}
                             items={timezones}
                             isDisabled={this.state.isAutomaticTimezone}
-                            defaultValue={this.state.selectedTimezone}
                             value={this.state.selectedTimezone}
+                            onInputChange={selectedTimezone => this.setState({selectedTimezone})}
```

Comment:
Is this supposed to be a custom `onChange` callback? Won't this override the `onInputChange` callback passed by Form?

If this is a custom `onChange`, can we also clean up [this hack](https://github.com/Expensify/App/blob/71a989b8a84405330aabf5bfac73de481848860e/src/pages/settings/Profile/ProfilePage.js#L171-L175) introduced in the `validate` method?